ci_get_latest_terraform.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Downloads the latest release of terraform for linux in the current working directory
+set -euxo pipefail
+
+terraform_latest_version=`curl -s "https://api.github.com/repos/hashicorp/terraform/releases/latest" | grep -o "tag_name\": \"[^\"]*\"" | grep -Po "(\d|\.)+"`
```

Comment:
What are your thoughts if we made this line with this:
```
curl -s "https://api.github.com/repos/hashicorp/terraform/releases/latest" | grep "tag_name" | grep -Po "(\d|\.)+"
```
Also, bash scripts are incorporating  this `$()` method rather than ` `` `

In the end we can use have this used. 

```
terraform_latest_version=$(curl -s "https://api.github.com/repos/hashicorp/terraform/releases/latest" | grep "tag_name" | grep -Po "(\d|\.)+")
```